src/index.ts

```diff
@@ -36,19 +37,76 @@ export {HttpRequest};
 export {detectServiceContext};
 
 const PKG = require('../../package.json');
-const v2 = require('./v2');
+import * as v2 from './v2';
 
-import {Entry} from './entry';
-import {Log, GetEntriesRequest, MonitoredResource, Severity, SeverityNames} from './log';
+import {Entry, LogEntry} from './entry';
+import {Log, GetEntriesRequest, LogOptions, MonitoredResource, Severity, SeverityNames} from './log';
 import {Sink} from './sink';
 import {Duplex} from 'stream';
 import {AbortableDuplex} from '@google-cloud/common';
+import {google} from '../proto/logging';
+import {google as google_config} from '../proto/logging_config';
+import {CallOptions} from 'google-gax/build/src/gax';
```

Comment:
This *should* be directly exported from `google-gax`.  If it isn't - can you please file a bug over in googleapis/gax-nodejs?

Reply:
My bad, import from `google-gax` works fine, thanks.